pkg/subscriptions/subscriptions.html

```diff
@@ -0,0 +1,157 @@
+                </td>
+              </tr>
+              <tr>
+                <td translatable="yes">User name</td>
+                <td>
+                  <input class="form-control" id="subscription-register-username" type="text"/>
+                </td>
+              </tr>
+              <tr>
+                <td translatable="yes">Password</td>
+                <td>
+                  <input class="form-control" type="password" id="subscription-register-password" value=""/>
+                </td>
+              </tr>
+            </table>
+          </div>
+          <div class="modal-footer">
+            <button class="btn btn-default" translatable="yes" data-dismiss="modal">
+              Cancel
+            </button>
+            <button class="btn btn-primary" id="account-register-start" translatable="yes">
+              Register
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <script id="subscription-template" type="x-template/mustache">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span>Installed Product ({{product_name}})</span>
+        </div>
+        <div class="panel-body">
+          <table class="cockpit-form-table">
+            <tbody>
+              <tr><td>Product name</td><td><span>{{product_name}}</span></td></tr>
+              <tr><td>Product ID</td><td><span>{{product_id}}</span></td></tr>
+              <tr><td>Version</td><td><span>{{version}}</span></td></tr>
+              <tr><td>Architecture</td><td><span>{{arch}}</span></td></tr>
+              <tr><td>Status</td><td><span>{{status}}</span></td></tr>
+              <tr><td>Starts</td><td><span>{{starts}}</span></td></tr>
+              <tr><td>Ends</td><td><span>{{ends}}</span></td></tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </script>
+
+    <div id="subscriptions-subscribed" class="container-fluid" hidden>
+      <div class="panel panel-default" id="subscription-system-info" hidden>
+        <div class="panel-heading">
+          <span>System</span>
+        </div>
+        <div class="panel-body">
+          <table class="cockpit-form-table">
+            <tbody>
+              <tr><td>System identity</td><td><span id="subscription-system-identity"></span></td></tr>
+              <tr><td>name</td><td><span id="subscription-system-name"></span></td></tr>
+              <tr><td>org name</td><td><span id="subscription-system-org-name"></span></td></tr>
+              <tr><td>org id</td><td><span id="subscription-system-org-id"></span></td></tr>
+              <tr><td>org name</td><td><span id="subscription-system-org-name"></span></td></tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+
+    <script>
+        require([
+        "subscriptions/subscriptions"
+        ], function(subscriptions) {
+            //
```

Comment:
Including an AMD package should not have side-effects like this. Maybe add an subscriptions.init() function and call it here?